src/components/AttachedActivity.test.tsx

```diff
@@ -18,9 +18,11 @@ const activity = {
   object: 'Random string',
 };
 
+type Activity = ComponentProps<typeof AttachedActivity>['activity'];
```

Comment:
can't you just do this instead?
```suggestion
type Activity = AttachedActivityProps['activity'];
```

Reply:
![image](https://user-images.githubusercontent.com/43254280/110475428-2ca5ea80-80e1-11eb-8269-be51c49eccb9.png)